### ISSUE TITLE
[akash provider run] --bid-deposit isn't working

### ISSUE THREAD

**Issue by arno01** [ovrclk/akash#1376]
versions tried:
- v0.12.1
- 0.12.2-7-gbd78206
- 0.12.2-8-g4a5dc5c

The only way to make it work was to rebuild the akash binary with this change:

```
akash$ git diff
diff --git a/x/market/types/params.go b/x/market/types/params.go
index 5a5c2a7..eedbd29 100644
--- a/x/market/types/params.go
+++ b/x/market/types/params.go
@@ -9,7 +9,7 @@ import (
 var _ paramtypes.ParamSet = (*Params)(nil)
 
 var (
-       DefaultBidMinDeposit        = sdk.NewCoin("uakt", sdk.NewInt(50000000))
+       DefaultBidMinDeposit        = sdk.NewCoin("uakt", sdk.NewInt(5000000))
        defaultOrderMaxBids  uint32 = 20
        maxOrderMaxBids      uint32 = 500
 )
```

Discord discussion
https://discord.com/channels/747885925232672829/784452434604982283/888526827621060629

**Comment by dmikey** [comment]
fixed by https://github.com/ovrclk/akash/pull/1375

**Comment by boz** [comment]
We should add the fix to the next mainnet release as well.

**Comment by hydrogen18** [comment]
This should be fixed in v0.14.0.
